src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -216,23 +231,34 @@ protected void end() {
 
     public Command align() {
         return new Command("align", this) {
-
-            NetworkTableInstance inst = NetworkTableInstance.getDefault();
-            NetworkTable table = inst.getTable("Vision Correction Table");
             double visionCorrectionFactor = table.getEntry("Vision Correction").getDouble(0);
+            boolean visionConfirmation = table.getEntry("Vision Found").getBoolean(false);
+            double visionTurnSpeed;
 
             @Override
             protected void execute() {
                 visionCorrectionFactor = table.getEntry("Vision Correction").getDouble(0);
-                drive.arcadeDrive(0, visionCorrectionMultiplier * visionCorrectionFactor);
+                visionConfirmation = table.getEntry("Vision Found").getBoolean(false);
+                // drive.arcadeDrive(0, visionCorrectionMultiplier * visionCorrectionFactor);
+                if ((visionCorrectionFactor > driveDeadband) && (visionConfirmation == true))
```

Comment:
`if(x == true)` is the same as `if(x)`, so you can just use that

Reply:
changed to ((visionCorrectionFactor < -driveDeadband) && (visionConfirmation))

Reply:
You don't need the parenthesis around a single variable, but it's not worth going back to fix unless you're changing the file again anyways.

Reply:
just && visionConfirmation)?